Makefile

```diff
@@ -66,3 +75,10 @@ image-release:
 .PHONY: push-release
 push-release:
 	docker push $(IMAGE):$(VERSION)
+
+generate-kustomize: bin/helm
```

Comment:
So this generates the kustomize from the helm charts, which are two distinct installation mechanisms, correct?  And after making a change to the helm chart, is it expected for a contributor to run this makefile target?